src/components/calcite-modal/calcite-modal.tsx

```diff
@@ -0,0 +1,195 @@
+import {
+  Component,
+  Element,
+  Prop,
+  Host,
+  Event,
+  EventEmitter,
+  Listen,
+  h,
+  Method,
+  State
+} from "@stencil/core";
+import { x24 } from "@esri/calcite-ui-icons";
```

Comment:
Same issue I have with alerts, when built this will include all icons. We can probably just set path data per component for now until we figure this out.